masterdata/configuration/concepts/defaultFormConcepts.csv

```diff
@@ -53,4 +53,6 @@ Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Descript
 9bb0795c-4ff0-0305-1990-000000000049,,,Referral Letter,Referral Letter,,,,,Misc,N/A,,,9bb0795c-4ff0-0305-1990-000000000042;164359AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA;9bb0795c-4ff0-0305-1990-000000000043;9bb0795c-4ff0-0305-1990-000000000044,,,,,,,,
 8b5901e0-67b7-4e56-ab00-69d60e929208,,,Types of Allergy,Allergy,,,,,Misc,Coded,,"Allergy;Food Allergy;Laboratory Animal Dander Allergy;Gastrointestinal Food Allergy;erythromycin allergy;naproxen allergy;house dust mite allergy;Allergy to Latex;heparin allergy;calcium channel blocker allergy;Personal History of Allergy to Anti-Infective Agent;Allergy-Sensitivity to Fungi Syndrome;Drug allergy;Dander (Animal) Allergy;Contrast Media Allergy;Allergy list;Allergy medication list;History of Allergy to Milk Products;Adverse Reaction to Sulfadoxine-Pyrimethamine;Grade of adverse reaction;Personal History of Allergy to Shellfish;allergy to wheat;allergy to sufentanil;allergy to sevoflurane;allergy to remifentanil;allergy to meropenem;allergy to imipenem;allergy to ertapenem;allergy to egg protein;allergy to dairy product;allergy to chloroprocaine;allergy to almonds;Personal History of Analgesic Agent Allergy;Personal History of Allergy to Unspecified Medicinal Agent;Allergy to drug;sulfite allergy;Allergy to Tomatoes;Hymenoptera Allergy;Allergy to Cats;History of Allergy;Allergy to Dog;Allergy to Insect Bites;Allergy to Insects;Allergy to Spiders;Allergy to Eggs;Allergy to Peanuts;Allergy to penicillin;Allergy to Seafood;Personal History of Allergy to Sulfonamides;Allergy, Unspecified;Personal History of Allergy to Anesthetic Agent;5-hydroxytryptamine-3 receptor antagonist allergy;4-quinolones allergy;Photoallergy",,,,,,,,,,
 5e67d33c-95fc-4122-ba9b-98a3adf57de0,,,Symptoms of The Reaction,Symptoms of The Reaction,,,,,Misc,Text,,,,,,,,,,,,
-19aac1a6-78ca-4302-a7cc-742d6af1dfa9,,,Allergies,Allergies,,,,,Misc,N/A,,,Types of Allergy;Symptoms of The Reaction;,,,,,,,,,
\ No newline at end of file
+19aac1a6-78ca-4302-a7cc-742d6af1dfa9,,,Allergies,Allergies,,,,,Misc,N/A,,,Types of Allergy;Symptoms of The Reaction;,,,,,,,,,
+3ed880f1-9164-4eb6-8e60-cee1c51606e1,,,Wax dissolving solution,,,,,,Drug,N/A,,,,,,,,,,,
```

Comment:
Dont add drugs related concepts in this CSV. This CSV is only for Forms related concepts.
Create a new CSV file for drugs.